auto_install/install.sh

```diff
@@ -2706,13 +2708,34 @@ confUnattendedUpgrades(){
 			fi
 		fi
 	elif [[ "${PKG_MANAGER}" == 'apk' ]]; then
-		echo 'https://dl-cdn.alpinelinux.org/alpine/edge/testing/' | ${SUDO} tee -a /etc/apk/repositories && ${SUDO} apk -q update
+		local down_dir
+		## install dependencies
+		# shellcheck disable=SC2086
+		${SUDO} ${PKG_INSTALL} unzip
 
-		PIVPN_DEPS=(apk-autoupdate)
+		if ! down_dir="$(mktemp -d)"; then
+			echo "::: Failed to create download directory for apk-autoupdate!"
+			exit 1
+		fi
 
-		installDependentPackages PIVPN_DEPS[@]
+		## download binaeries
```

Comment:
typo! :p 

this one just jumped to my eyes (i am usually not good at getting typos, and do a lot of them myself! :p ): binaries